### ISSUE TITLE
Change Process Name

### ISSUE THREAD

**Issue by ialfarol** [sw3103/movemouse#62]
Hi,

Is there a way to change the process name or instance name of this beautiful and useful app? I got a second job and they use a shitty software for productivity and time-keeping, but also keeps track of open applications in the computer, so obviously wouldn't like to see in a report Movemouse.exe running haha

**Comment by MindShaver** [comment]
Open it up in Visual Studio, right click on the project and click "Properties". Change the Assembly name to something generic. This will change the process name. I also went through the project and replaced all instances of "Mouse" that I could with something generic.

**Comment by ialfarol** [comment]
> Open it up in Visual Studio, right click on the project and click "Properties". Change the Assembly name to something generic. This will change the process name. I also went through the project and replaced all instances of "Mouse" that I could with something generic.

Hey there, I feel completely dumb as my developer skills are close to 0. I tried opening the .exe file, no luck, then tried cloning the Github repo and I managed to change the instance name, but I cannot find any way to compile/publish it back to an .exe file :(

**Comment by MindShaver** [comment]
No worries at all. I'm not sure how you are running MoveMouse but this is how I do it -

- I have Visual Studio Community installed
- I cloned the MoveMouse repository to my local machine
- I open the MoveMouse.sln with Visual Studio
- I click the run button and it runs MoveMouse locally with no need for any internet connection or anything else

If that is how you're running it also then in Visual Studio you can right click on the MoveMouse project name, click properties, and rename the assembly.

If that doesn't help I can try to fork the repository and make that change for you or send you better detailed instructions.

**Comment by sw3103** [comment]
Hi @ialfarol,

Did you download Move Mouse from GitHub, or install it through the Microsoft Store?

If you downloaded it directly from GitHub, can you not simply rename the "Move Mouse.exe" before launching it?

This option would not be possible with the version installed through the Microsoft Store.

Also - take a look in the Appearance tab to see the various other options that might help obfuscate it.

https://github.com/sw3103/movemouse/wiki#appearance

**Comment by sw3103** [comment]
Hi @ialfarol,

Was my suggestion any good to you?

Please let me know if you need any further assistance.

**Comment by sw3103** [comment]
Closing due to inactivity.

**Comment by ialfarol** [comment]
Hey! Sorry I missed these emails. So I did try with another app just renaming the .exe file but looks like this is not the value this crappy Hubstaff thing is reading. I tried Caffeine for practical purposes in case if failed, and didn't want to show MoveMouse as my go-to option haha, and renamed to sysdeck64.exe, but see what the app pulls:
![Screenshot 2024-02-16 at 8 50 50 AM](https://github.com/sw3103/movemouse/assets/156240719/29546bd3-2fdd-4c30-910f-57fe8fbf063f)
